tests/flow/test_function_calls.py

```diff
@@ -227,12 +228,79 @@ def test10_modulo_inputs(self):
         expected_result = [[0]]
         self.env.assertEquals(actual_result.result_set, expected_result)
 
-        # Validate modulo by 0
+        # Validate integer dividend modulo by 0
+        # redis-cli output example:
+        # 127.0.0.1:6379> GRAPH.QUERY g "RETURN 3 % 0"
+        # (error) Division by zero
         query = "RETURN 3 % 0"
         try:
             actual_result = graph.query(query)
         except redis.ResponseError as e:
             self.env.assertContains("Division by zero", str(e))
+        
+        # Validate floating-point dividend modulo by 0
+        # redis-cli output example:
+        # 127.0.0.1:6379> GRAPH.QUERY g "RETURN 1.0 % 0"
+        # 1) 1) "1.0 % 0"
+        # 2) 1) 1) "-nan"
+        query = "RETURN 1.0 % 0"
+        actual_result = graph.query(query)
+        self.env.assertTrue(math.isnan(actual_result.result_set[0][0]))
+
+        # Validate integer dividend modulo by 0.0
+        # redis-cli output example:
+        # 127.0.0.1:6379> GRAPH.QUERY g "RETURN 1 % 0.0"
+        # 1) 1) "1 % 0.0"
+        # 2) 1) 1) "-nan"
+        query = "RETURN 1 % 0.0"
+        actual_result = graph.query(query)
+        self.env.assertTrue(math.isnan(actual_result.result_set[0][0]))
```

Comment:
from `man fmod`

```
SPECIAL VALUES
     fmod(±0, y) returns ±0 if y is neither 0 nor NaN.

     fmod(x, y) returns a NaN and raises the "invalid" floating-point exception for x infinite or y zero.

     fmod(x, ±infinity) returns x for x not infinite.
```

Reply:
I'll add test cases for those special values.
Is there a constant to represent infinite? I could test like this:
```
127.0.0.1:6379> GRAPH.QUERY g "RETURN 0 % ( 1 / 0.0 )"
1) 1) "0 % ( 1 / 0.0 )"
2) 1) 1) "0"
```
```
127.0.0.1:6379> GRAPH.QUERY g "RETURN 0.0 % ( 1 / 0.0 )"
1) 1) "0.0 % ( 1 / 0.0 )"
2) 1) 1) "0"
```

But it depends on division correctness.

Reply:
Tests were added in [1572385](https://github.com/RedisGraph/RedisGraph/pull/2585/commits/1572385051c22c77613807e9987fb3b47e44ec41)